ynr/apps/elections/uk/templates/candidates/_person_form.html

```diff
@@ -95,12 +95,16 @@ <h2>Candidacy:</h2>
       <thead>
         <th>Ballot</th>
         <th>Party</th>
+        <th>Previous Party Affiliation</th> 
```

Comment:
I'm not sure we should have this as its own header/column in the table, as it means it is going to show for every candidate when for 99% of candidates it is will have nothing to display. It might be better to include easier to include it in the `<td>` element under the party name? A bit more like how we display on the person page

Reply:
This is what I mean by it will display unnecessarily
![Screenshot 2022-03-23 at 16 18 28](https://user-images.githubusercontent.com/15347726/159746569-fbbcc43d-dea1-433b-86a4-a3961be6c60f.png)